packages/pwa-kit-react-sdk/src/ssr/universal/utils.js

```diff
@@ -51,3 +51,64 @@ export const getProxyConfigs = () => {
     // Clone to avoid accidental mutation of important configuration variables.
     return configs.map((config) => ({...config}))
 }
+
+/**
+ * Returns the express app configuration file in object form. The object resolution will
+ * be as follows (from highest to lowest priority):
+ *
+ * > {target_name}.ext
+ * > local.ext
+ * > default.ext
+ * > package.json (mobify.key)
+ *
+ * Each file marked with `ext` can optionally be terminated with `yml`, `yaml` or
+ * `json` in that priority.
+ *
+ * NOTE: This is an isomorphic function, when run on the browser the config returned will
+ * be the value serialized in the html.
+ *
+ * @returns - the application configuration object.
+ */
+/* istanbul ignore next */
+export const getConfig = () => {
+    if (onClient) {
+        return (
+            window.__CONFIG__ ||
+            JSON.parse(document.getElementById('mobify-data').innerHTML).__CONFIG__
+        )
+    }
+
+    const isRemote = Object.prototype.hasOwnProperty.call(process.env, 'AWS_LAMBDA_FUNCTION_NAME')
+    let moduleName = process?.env?.DEPLOY_TARGET || ''
```

Comment:
does it make sense to add an argument to `getConfig` for moduleName and fallback to `process?.env?.DEPLOY_TARGET`?

Reply:
I think having an argument would make the function less universal in a way. It would be an argument that only makes sense on the server. That is why I shy'd away from it. With regard to the upload script I do set the environment variable based on the target argument if one exists.